app/config.py

```diff
@@ -422,6 +422,8 @@ def sl_getenv(env_var: str, default_factory: Callable = None):
 ZENDESK_API_TOKEN = os.environ.get("ZENDESK_API_TOKEN")
 ZENDESK_ENABLED = "ZENDESK_ENABLED" in os.environ
 
+ENABLE_DMARC_CHECK = os.environ.get("ENABLE_DMARC_CHECK")
```

Comment:
for flag, usually we use something like `ENABLE_DMARC_CHECK = "ENABLE_DMARC_CHECK" in os.environ` I think

Reply:
Changed.